File: Scripts/Listener/ILoginListener.cs
// *******************************************
// Company Name:	深圳市晴天互娱科技有限公司
//
// File Name:		ILoginListener.cs
//
// Author Name:		Bridge
//
// Create Time:		2024/09/03 10:36:11
// *******************************************

namespace Bridge.Common
{
	/// <summary>
	/// 
	/// </summary>
	public interface ILoginListener
	{
		void OnSuccess(string accessToken);

		void OnCancel();

		void OnError(int errCode, string errMsg);
	}
}

File: Plugins/Android/ThirdSDK.androidlib/src/main/java/com/bridge/common/listener/ILoginListener.java
package com.bridge.common.listener;

public interface ILoginListener {
    /**
     * 分享成功
     */
    void onSuccess(String accessToken);

    /**
     * 用户取消登录
     */
    void onCancel();

    /**
     * 分享失败
     *
     * @param errCode 错误码
     * @param errMsg  错误信息
     */
    void onError(int errCode, String errMsg);
}
